addons/website/views/snippets/snippets.xml

```diff
@@ -147,6 +147,7 @@
                 <t t-snippet="website.s_quotes_carousel" string="Quotes" group="people">
                     <keywords>testimonials</keywords>
                 </t>
+                <t t-snippet="website.s_company_team_shapes" string="Team Shapes" group="people"/>
```

Comment:
I think we can add some keywords so it can be found more easily during the the search. I suggest:
`<keywords>organization, structure, people, team, name, role, position, image, portrait, photo, employees, shapes</keywords>`